lib/windows/episodes.py

```diff
@@ -513,10 +513,11 @@ def optionsButtonClicked(self, from_item=False):
             if len(mli.dataSource.media) > 1:
                 options.append({'key': 'play_version', 'display': T(32451, 'Play Version...')})
 
-            if mli.dataSource.isWatched and not mli.dataSource.viewOffset.asInt():
-                options.append({'key': 'mark_unwatched', 'display': T(32318, 'Mark Unplayed')})
-            else:
+            inProgress = mli.dataSource.viewOffset.asInt()
+            if not mli.dataSource.isWatched or inProgress:
                 options.append({'key': 'mark_watched', 'display': T(32319, 'Mark Played')})
+            if mli.dataSource.isWatched or inProgress:
```

Comment:
This should just be an `else:`

The correct behavior for an item that is unplayed or played but in progress is to mark watched - other was mark unwatched.

Reply:
Hm no, that's what this PR is about. PlexWeb now shows both Mark Played/Unplayed for in progress items (which makes sense).

Reply:
(and it's always bugged me that that wasn't the case before)